tools/webdriver/webdriver/transport.py

```diff
@@ -104,9 +104,12 @@ def connection(self):
             conn_kwargs = {}
             if self._timeout is not None:
                 conn_kwargs["timeout"] = self._timeout
-
-            self._conn = httplib.HTTPConnection(
-                self.host, self.port, strict=True, **conn_kwargs)
+            if PY3:
```

Comment:
Can't we just write this

```py
if not PY3:
     conn_kwargs["strict"] = True
self._conn = HTTPConnection(self.host, self.port, **conn_kwargs)
```

Which minimises the divergence between the two codepaths.

Reply:
Yes, it's a better solution. Updated the code. Thanks!